snowshu/core/configuration_parser.py

```diff
@@ -108,10 +108,10 @@ class ConfigurationParser:
     default_case: str = None
     preserve_case: bool = False
 
-    def _get_dict_from_anything(self,
-                                dict_like_object: Union[str, 'StringIO', dict],
-                                schema_path: Path) -> dict:
-        """Returns dict from path, io object or dict.  
+    def get_dict_from_anything(self,
```

Comment:
Is this only moving from public to private due to its use in the new adapter?

Reply:
That's correct, I guess it's correlated with the other question regarding import of ConfigurationParser(), so if we solve the other one we could revert this change.